### ISSUE TITLE
Transaction comment is ignored while writing transaction to history file

### ISSUE THREAD

**Issue by kaankalendr** [symstore/symstore#23]
While writing transaction to history file, transaction comment is being skipped due to missing variable in `__str__` method in `symstore/symstore.py`.

Problem method (`starts with line 286 in symstore/symstore.py`):
```python
    def __str__(self):
        date_stamp = self.timestamp.strftime("%m/%d/%Y")
        time_stamp = self.timestamp.strftime("%H:%M:%S")

        assert self._commited()
        return """%s,%s,%s,%s,%s,"%s","%s","",""" % \
               (self.id, self.type, self.ref, date_stamp, time_stamp,
                self.product, self.version)
```

Possible solution:
```python
    def __str__(self):
        date_stamp = self.timestamp.strftime("%m/%d/%Y")
        time_stamp = self.timestamp.strftime("%H:%M:%S")

        assert self._commited()
        return """%s,%s,%s,%s,%s,"%s","%s","%s",""" % \
               (self.id, self.type, self.ref, date_stamp, time_stamp,
                self.product, self.version, self.comment)
```

**Comment by elmjag** [comment]
Writing transaction comments is basically not supported at the moment.

There is a bit of work that needs to be done, for example the command line utility needs some option for specifying comments. I'll take a look in not to distant future, as my schedule permits.

**Comment by kaankalendr** [comment]
I am using comments in some clean up scripts and I am adding a command like below before.
`cmd.exe /c %SYMSTORE% add /s %SYMBOL_SERVER% /compress /r /f %EXE_FILES% /t %PROJECT% /v %VERSION% /c %COMMENT%`

It would be great having comments also supported with this library but I can figure out a work around for now. Thanks for your reply.

**Comment by elmjag** [comment]
I have implemented new '--comment' option https://github.com/symstore/symstore/commit/31058003c8bcd0cb612b174e9996b4196cb53bf5

I'm planning to make a new release with that feature soon. In the meantime you can give it a go, if you are ok with installing the package from source.

**Comment by elmjag** [comment]
--comment option is now available in v0.3.4
